src/main/java/picard/analysis/TheoreticalSensitivity.java

```diff
@@ -143,7 +143,7 @@ We use an O(1) stochastic acceptance algorithm -- see Physica A, Volume 391, Pag
         private Random rng;
 
         RouletteWheel(final double[] weights) {
-            rng = new Random(51);
+            rng = new Random(randomSeed);
```

Comment:
I suspect you are referring to the Utils in GATK4.  Unfortunately this is Picard :(